espflash/src/targets/esp32c2.rs

```diff
@@ -95,11 +97,11 @@ impl Target for Esp32c2 {
     ) -> Result<IdfBootloaderFormat<'a>, Error> {
         let booloader: &'static [u8] = match xtal_freq {
             XtalFrequency::_40Mhz => {
-                println!("Using 40MHz bootloader");
+                debug!("Using 40MHz bootloader");
```

Comment:
Nice, I've been meaning to make an issue on that :D

Reply:
It was never meant to be released this way, but it slipped in